docs/markdown/install/training.md

```diff
@@ -1,3 +1,54 @@
 # Training Library Installation
 
-TODO.
\ No newline at end of file
+## Prerequisites
+* [anaconda3](https://www.anaconda.com/products/individual)
```

Comment:
emmm, I choose conda enviorment because it provide CUDNN and Cudatoolkit virtual environment using just "conda install", or our user have to install these from scratch, which is hard to compat with the cuda version.